packages/alice/lib/core/alice_utils.dart

```diff
@@ -2,7 +2,7 @@ import 'package:flutter/foundation.dart' show kReleaseMode, debugPrint;
 
 /// Utils used across multiple classes in app.
 class AliceUtils {
-  static void log(String logMessage) {
+  static void log(dynamic logMessage) {
```

Comment:
`debugPrint` can only print a `String`. Also, why would you want to log a `dynamic`?